### DISCUSSION TITLE
real time torque value

### DISCUSSION THREAD

**Discussion by 7enHom** [bitcraze/discussions#1663] [Q&A]
Sorry to interrupt.
If I want to get the real time torque value of the crazyflie2.1, is there any way to do that? Thank you！

**Comment by ataffanel** [comment]
Crazyflie does not have any way to measure torque of the motors, so the torque can only be estimated and not measured directly. As far as we are aware, no one has devised a formula yet for torque estimation on the Crazyflie. This would need to be done by measurements on a specialized testing rig.

**Comment by 7enHom** [comment]
OK, Thanks.
